virttest/utils_libvirt/libvirt_keywrap.py

```diff
@@ -0,0 +1,41 @@
+"""
+Module to contain logic testing protected key module 'pkey'
+"""
+import logging
+import os
+
+from virttest.utils_misc import cmd_status_output
+
+
+class ProtectedKeyHelper(object):
+    """
+    Helper class to test for protected key support on s390x
+    """
+    def __init__(self, session=None):
+        self.session = session
+        self.sysfs = "/sys/devices/virtual/misc/pkey/protkey"
+        self.module_name = "pkey"
+
+    def load_module(self):
+        """
+        Loads pkey module
```

Comment:
Could you add a new blank line under this line? And same to the following functions.